ingesters/HttpIngester/config.go

```diff
@@ -37,13 +39,15 @@ const (
 
 type gbl struct {
 	config.IngestConfig
-	Bind                    string
-	Max_Body                int
-	TLS_Certificate_File    string
-	TLS_Key_File            string
-	Health_Check_URL        string
-	Max_Connections         int
-	Max_Concurrent_Requests int
+	Bind                       string
+	Max_Body                   int
+	TLS_Certificate_File       string
+	TLS_Key_File               string
+	Health_Check_URL           string
+	Max_Connections            int
+	Max_Concurrent_Requests    int
+	Timestamp_Max_Past_Delta   string
```

Comment:
I don't believe we should have one offs in these, I think it should be added to the `config.IngestConfig` so that we can use them in all ingesters more easily.

As is we are going to have to make changes to every single of our ingesters and every single listener.

We already have the timegrinder override stuff that timegrinder can pick up using our abstracted functions, I think this should go to that same place using that same methodology